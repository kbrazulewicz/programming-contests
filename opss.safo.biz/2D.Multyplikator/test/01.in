10
10 34 54 23 65 87 45 63 27 87 0
10 12 12 12 12 12 12 12 12 12 12
10 45 45 45 45 45 45 45 45 45 45
10 20 20 20 20 20 20 20 20 20 20
10 666 666 666 666 666 666 666 666 666 666
6 6 66 6 66 6 66
6 666 666 666 666 666 666
2 999 999
3 666 777 888
10 1 1 1 1 1 1 1 1 1 1
