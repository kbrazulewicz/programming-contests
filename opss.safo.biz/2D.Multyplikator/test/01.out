1
11
17
14
29
8
17
6
9
1
